content/blog/2018-02-07-strict-mode.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Introducing Strict Mode
+author: [bvaughn]
+---
+
+`StrictMode` is a tool for highlighting potential problems in an application. Like `Fragment`, `StrictMode` does not render any visible UI. It simply activates additional checks and warnings for its descendants.
+
```

Comment:
Can we add a paragraph reminding people how deprecations work and explaining how strict mode fits in?

* remember how react versions and warnings work?
* what if you want to get ahead of the curve (either because you're awesome or for greenfield code in an existing app)
* strict mode helps you do that and gives you warnings from the future, sooner

I guess the double lifecycle thing is not planned to be global though? Or are we doing that in 17 too? I missed that.

Reply:
Double lifecycle is specific to strict mode. I don't think we have any plans to turn this on outside fo strict mode.

Reply:
Put another way, strict mode exists for more than just advanced notification of deprecations.